Jetcaster/tv-app/src/main/java/com/example/jetcaster/tv/ui/component/thumbnailPlaceholder.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jetcaster.tv.ui.component
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.painter.BrushPainter
+import androidx.tv.material3.ExperimentalTvMaterial3Api
+import androidx.tv.material3.MaterialTheme
+
+@OptIn(ExperimentalTvMaterial3Api::class)
+@Composable
+internal fun thumbnailPlaceholder(
```

Comment:
Thoughts about adding this to `designsystem` module?

Reply:
I would, and it would be tricky as it refers to the `MaterialTheme` object from `tv-material` library. 

There are three possible options for this:

1. Leave the component in `tv-app` module
2. Remove the default parameter value from the function
3. Change the default parameter value to Color.Transparent from MaterialTheme.colorScheme.backgroundVariant

I would take the 3rd option to move the function to the `design` module. What do you think?